quarkus-cli/src/test/java/io/quarkus/ts/quarkus/cli/QuarkusCliExtensionsIT.java

```diff
@@ -111,18 +115,25 @@ private void whenGetListExtensions(String... extraArgs) {
         assertResultIsSuccessful();
     }
 
-    private void assertListOriginsOptionOutput() {
-        // Origins only shows extension name ++ version. Reported by https://github.com/quarkusio/quarkus/issues/18062.
+    private void assertListDefaultOptionOutput() {
         assertTrue(result.getOutput().contains(AGROAL_EXTENSION_NAME)
-                && !result.getOutput().contains(AGROAL_EXTENSION_ARTIFACT)
+                && result.getOutput().contains(AGROAL_EXTENSION_ARTIFACT)
+                && !result.getOutput().contains(AGROAL_EXTENSION_GUIDE),
+                "Default output is unexpected. Output: " + result.getOutput());
+    }
+
+    private void assertListOriginsOptionOutput() {
+        assertTrue(result.getOutput().contains(QUARKUS_BOM)
+                && result.getOutput().contains(AGROAL_EXTENSION_NAME)
+                && result.getOutput().contains(AGROAL_EXTENSION_ARTIFACT)
                 && !result.getOutput().contains(AGROAL_EXTENSION_GUIDE),
                 "--origins option output is unexpected. Output: " + result.getOutput());
     }
 
     private void assertListNameOptionOutput() {
         // Concise shows only the artifact id
         assertTrue(result.getOutput().contains(AGROAL_EXTENSION_ARTIFACT)
-                && !result.getOutput().contains(AGROAL_EXTENSION_NAME),
+                && result.getOutput().contains(AGROAL_EXTENSION_NAME),
```

Comment:
According to the line 134 `Concise shows only the artifact id`
Isn't this a bug?

Reply:
Checks are relevant, things changed around extension listing. That comment is no longer relevant, I will create new PR to do some polish, I will include it there.